tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -18,7 +18,9 @@ const { getSharedWithOthersFiles, getSharedWithOthersFolders } = require('../hel
 
 const assertDesiredResourcesListed = async function (resourcesMatchingPattern, listedResources) {
   const diff = _.difference(resourcesMatchingPattern, listedResources)
-  return assert.strictEqual(diff.length, 0, `Expected : ${resourcesMatchingPattern} but got ${listedResources}`)
+  const difference = _.difference(listedResources, resourcesMatchingPattern)
```

Comment:
give it a better name, to confusing with `diff`

Reply:
fixed